sdk/remoterendering/mixed-reality-remote-rendering/src/lro/renderingSessionPoller.ts

```diff
@@ -147,7 +147,13 @@ export class RenderingSessionPoller extends Poller<
         new RenderingSessionOperationStateImpl(renderingSession)
       )
     );
-    this.intervalInMs = options.intervalInMs ? options.intervalInMs : 10000;
+    if (options.intervalInMs) {
+      this.intervalInMs = options.intervalInMs;
+    } else if (renderingSession.size === KnownRenderingServerSize.Standard) {
```

Comment:
nit: could we add a comment here why this is done? Something like "Standard session startup time is now in the range of x-y s. Because of this we want to poll more frequently in order to detect a ready session with less delay. "